server/common/rest.py

```diff
@@ -409,3 +416,64 @@ def summarize_var_post(request, data_adaptor):
 
     key = request.args.get("key", default=None)
     return summarize_var_helper(request, data_adaptor, key, request.get_data())
+
+
+def spatial_image_get(request, data_adaptor):
+    """
+    Retrieve a spatial image from the data adaptor and return it as a PNG file
+    """
+
+    resolution = "hires"
+
+    spatial = data_adaptor.get_spatial()
+    response_image = io.BytesIO()
+    img = spatial[resolution]
+    plt.imsave(response_image, img)
+    response_image.seek(0)
+    library_id = "test_library_id"
+
+    try:
```

Comment:
should we try to use JPEG here 👌 ?

I have the code in this ticket!

https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/gh/chanzuckerberg/single-cell-explorer/797